infra/label_suggestion_step_functions/lambdas/suggest_labels.py

```
@@ -0,0 +1,345 @@
+    logger.info(f"Downloaded {file_count} files to {cache_path}")
+    return cache_path
+
+
+async def process_batch(
+    receipts: List[Dict[str, Any]],
+    dynamo_client: Any,
+    chroma_client: Any,
+    embed_fn: Any,
+    llm: Any,
+    dry_run: bool = True,
+    batch_bucket: str = "",
```

Ruff (0.14.8)

Unused function argument: `batch_bucket`

(ARG001)